server/service/apple_mdm.go

```diff
@@ -5556,21 +5562,21 @@ func (d *digiCertVarsFound) SetPassword(value string) (*digiCertVarsFound, bool)
 	return d, !alreadyPresent
 }
 
-func isDigiCertConfigured(ctx context.Context, appConfig *fleet.AppConfig, ds fleet.Datastore,
+func isDigiCertConfigured(ctx context.Context, allDigiCertCAs []fleet.DigiCertCA, ds fleet.Datastore,
```

Comment:
Consider using `fleet.GroupedCertificateAuthorities` as the parameter type for this as well as `isNDESSCEPConfigured`  and `isCustomSCEPConfigured`